src/components/Panels/DeployContract.vue

```diff
@@ -455,9 +455,8 @@ export default {
         return {
           vname: item.vname,
           type:
-            validatedType.parsed !== validatedType.initial &&
-            validatedType.parsed === "ByStr20"
-              ? validatedType.parsed
+            validatedType && validatedType === "ByStr20"
+              ? validatedType
```

Comment:
can we do `getParamType(type) || type`?

Reply:
https://github.com/Zilliqa/neo-savant/issues/135

Should've used startsWith() instead of includes()